### PR TITLE
[POZ-2023] Adding talk details and updating sponsor logo

### PR DESCRIPTION
**Pull request by luprzybyl**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by luprzybyl** [comment]
Hi, May I ask to review and merge?

File: static/img/sponsors/allegro.png
Comment (adrianmoisey) [inline static/img/sponsors/allegro.png:1]: For updates to sponsors, we generally want to keep the previous change around, see https://github.com/devopsdays/devopsdays-web/blob/main/utilities/README.md#updating-a-sponsor-image-or-data

Do you know if all previous events should also get the new logo or should this change only apply to new events?

Comment (luprzybyl) [reply static/img/sponsors/allegro.png:1]: Hi, all previous events should also get the new logo. This is why I intentionally replaced the image. Company is the same, they just did a rebranding.

Comment (luprzybyl) [reply static/img/sponsors/allegro.png:1]: although, if this is your general preference, I can update new logo only for the upcoming edition.

Comment (luprzybyl) [reply static/img/sponsors/allegro.png:1]: I have updated logo as recommended

**Pull request merged**